app/views/posts/index.html.erb

```diff
@@ -1 +1,36 @@
-<h1>All posts by user</h1>
\ No newline at end of file
+<section class="main">
+  <header class="user">
+    <img src=<%= @user.photo %> alt="<%= @user.name %>'s photo">
+    <div class="container">
+      <h2><%= @user.name %></h2>
+      <p>Number of posts:
+        <%= @user.posts_counter %></p>
+    </div>
+  </header>
+  <% @user.posts.each_with_index do |post, index| %>
+      <article>
+        <div class="container">
+          <h3>Post #<%= index + 1 %></h3>
+          <p><%= post.text.truncate_words(10, omission: '...') %></p>
+          <div>
+            <span>Comments:
+              <%= post.comments_counter or '0' %></span>
+            <span>Likes:
+              <%= post.likes_counter or '0' %></span>
+          </div>
+        </div>
+        <% if post.comments.length != 0 %>
+          <ul class = "container">
+            <% post.comments.each do |comment| %>
+              <li><%= comment.author.name %>: <%= comment.text %></li>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use the `recent_comments` method you created in posts model here to only render the 5 most recent comments, Not all comments as you did.